content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -0,0 +1,291 @@
+---
+title: Unified Service Tagging
+kind: documentation
+aliases:
+- /tagging/unified_service_tagging
+further_reading:
+- link: "/tagging/using_tags"
+  tag: "Documentation"
+  text: "Learn how to use tags in the Datadog UI"
+- link: "/blog/autodiscovery-docker-monitoring"
+  tag: "Blog"
+  text: "Learn more about Autodiscovery"
+---
+
+## Overview
+Unified service tagging ties Datadog telemetry together through the use of three [reserved tags][1]: `env`, `service`, and `version`. By adding these tags to your [Datadog Agent's][2] configuration file, you can troubleshoot issues and view data pertaining to your services, based on environment or version, in a unified fashion within the Datadog app. Once these tags are configured you'll be able to do things like filter your logs, traces, and metrics by `service` and `version` to see the total service requests and the errors by version.
+
+### Requirements
+
+- Unified service tagging requires knowledge of configuring tags. If you are unsure of how to configure tags, read the [Getting Started with Tagging][3] guide and [Assigning tags][4] documentation before proceeding to configuration.
+
+- Unified service tagging requires the [Datadog Agent][5].
+
+{{< img src="tagging/unified_service_tagging/overview.gif" alt="Unified Service Tagging"  >}}
+
+## Configuration
+
+With unified service tagging, the `service` tag is configured with `env` and `version` at container or top-level to automatically apply these tags to all metrics, logs, and traces for all services. The Datadog Agent uses [environment variables][6], which are in `key:value` pairs, to configure tags for the Datadog app. In containerized environments using [Autodiscovery][7], the agent uses Kubernetes/Docker labels or environment variables to configure tags.
```

Comment:
What do we mean by the `key:value` pairs for the agent's environment variables? I think only one agent env var does that:
```
DD_TAGS = "key:value,key2:value2"
```

Maybe we should phrase this paragraph around quickly summarizing per environment the gist of how things work:
> Unified service tagging relies on configuring the `service`, `env`, and `version` tags.
> To properly configure unified tagging, ...

I think we can go into the details of what the container labels and agent config files are in the non-containerized and containerized sections.